providers/class.two-factor-email.php

```diff
@@ -39,8 +39,8 @@ function delete_token( $user_id ) {
 	function generate_and_email_token( $user ) {
 		$token = $this->generate_token( $user->ID );
 
-		$subject = sprintf( __( 'Your login confirmation code for %s', 'two-factor' ), get_bloginfo( 'name' ) );
-		$message = sprintf( __( 'Enter %s to log in.', 'two-factor' ), $token );
+		$subject = sprintf( esc_html__( 'Your login confirmation code for %s', 'two-factor' ), get_bloginfo( 'name' ) );
```

Comment:
Subject lines don't normally support html, so content shouldn't be escaped as such?